features-json/webtransport.json

```diff
@@ -22,7 +22,12 @@
     }
   ],
   "bugs":[
-    
+    {
+      "description": "There's [an issue](https://issues.chromium.org/issues/40069954) with excessive throttling of WebTransport connections in chromium browsers that cannot be mitigated by code."
+    },
+    {
+      "description": "[An issue](https://issues.chromium.org/issues/326887753?pli=1) in chromium with dropped messages."
```

Comment:
I attempted an updated description. let me know if that is sufficient. Essentially, the following points make things more clear:

1. [WebTransport spec](https://www.w3.org/TR/webtransport/#example-sending-stream) says that you can open a stream, send data, then close the stream, and have examples reading that data.
2. The reader side of this does *not* get all writes if you close the stream like this. 

This is likely a bug in either the spec, or chromium implementation. However, Firefox does the intuitive thing (working like node.js `.end()` on Response and Sockets) where it will ensure all data is sent prior to closing the write side.